type.go

```diff
@@ -20,3 +22,14 @@ var NewStringMap = nkngomobile.NewStringMap
 
 // NewStringMapWithSize creates an empty StringMap with a given size.
 var NewStringMapWithSize = nkngomobile.NewStringMapWithSize
+
+type wsConn interface {
```

Comment:
We should move it to `client.go`. This file is for gomobile compatibility

Reply:
Yes, modified.